Assets/NoesisGUI/Xaml/HUD.xaml

```diff
@@ -74,16 +74,24 @@
                 </LinearGradientBrush>
             </Border.Background>
         </Border>
-        <Border  BorderThickness="1" CornerRadius="0, 50, 0, 0" Grid.Column="1" Grid.ColumnSpan="3" Grid.Row="3"  Background="#3FD1D1D1" Margin="10" >
-
-        </Border>
 
         <Border x:Name="HEALTH_RECT"  BorderThickness="1" CornerRadius="0, 50, 0, 0" Grid.Column="1" Grid.ColumnSpan="3" Grid.Row="3"  Background="#FF00CF38" Margin="10" RenderTransformOrigin="0.0,0.0">
             <Border.RenderTransform>
                 <ScaleTransform ScaleY="1"/>
             </Border.RenderTransform>
         </Border>
 
+        <Border  BorderThickness="1" CornerRadius="0, 50, 0, 0" Grid.Column="1" Grid.ColumnSpan="3" Grid.Row="3" Margin="10" >
+            <Border.Background>
+                <LinearGradientBrush EndPoint="0.5,1" MappingMode="RelativeToBoundingBox" StartPoint="0.5,0">
+                    <GradientStop Color="Black"/>
+                    <GradientStop Color="#72FFFFFF" Offset="1"/>
+                    <GradientStop Color="#D6494949"/>
+                    <GradientStop Color="#7200CF38" Offset="0.496"/>
+                </LinearGradientBrush>
+            </Border.Background>
+        </Border>
+
         <TextBlock x:Name="HEALTH_DISPLAY"  Grid.Column="2" Grid.Row="3" Text="100%" FontSize="24" HorizontalAlignment="Center" VerticalAlignment="Center" Margin="0" Foreground="#FFC7144D" FontWeight="Bold" Height="31.923" Width="62.224" />
         <!--<Image Source="Test.png" Grid.Column="0" Grid.ColumnSpan="2" Grid.Row="0" Grid.RowSpan="2" Margin="0" Width="383.852" Height="392.727" VerticalAlignment="Bottom" HorizontalAlignment="Left"/>
         -->
```

Comment:
Dead code and comments should be avoid to have in master

Reply:
Done in a separate branch.